src/modules/cpu/kernel/lens_correction.hpp

```diff
@@ -0,0 +1,182 @@
+            for(; vectorLoopCount < alignedLength; vectorLoopCount += vectorIncrement)
+            {
+                // float z = 1./zCamera, x = xCamera*z, y = yCamera*z;
+                __m256 pZ = _mm256_div_ps(avx_p1, pZCamera);
+                __m256 pX = _mm256_mul_ps(pXCamera, pZ);
+                __m256 pY = _mm256_mul_ps(pYCamera, pZ);
+
+                // float xSquare = x*x, ySquare = y*y, r2 = xSquare + ySquare;
+                __m256 pX2 = _mm256_mul_ps(pX, pX);
+                __m256 pY2 = _mm256_mul_ps(pY, pY);
+                __m256 pR2 = _mm256_add_ps(pX2, pY2);
+
+                // float xyMul2 = 2*x*y;
+                __m256 p2xy = _mm256_mul_ps(avx_p2, _mm256_mul_ps(pX, pY));
+
+                // float kr = (1 + ((rCoeff[2]*r2 + rCoeff[1])*r2 + rCoeff[0])*r2)/(1 + ((rCoeff[5]*r2 + rCoeff[4])*r2 + rCoeff[3])*r2);
+                __m256 pNum = _mm256_fmadd_ps(_mm256_fmadd_ps(_mm256_fmadd_ps(pRCoeff[2], pR2, pRCoeff[1]), pR2, pRCoeff[0]), pR2, avx_p1);
+                __m256 pDen = _mm256_fmadd_ps(_mm256_fmadd_ps(_mm256_fmadd_ps(pRCoeff[5], pR2, pRCoeff[4]), pR2, pRCoeff[3]), pR2, avx_p1);
+                __m256 pKR = _mm256_div_ps(pNum, pDen);
+
+                // float colLoc = fx * (x * kr + tCoeff[0] * xyMul2 + tCoeff[1] * ( r2 + 2 * xSquare)) + u0;
+                __m256 pFac1 = _mm256_mul_ps(pX, pKR);
+                __m256 pFac2 = _mm256_mul_ps(pTCoeff[0], p2xy);
+                __m256 pFac3 = _mm256_mul_ps(pTCoeff[1], _mm256_fmadd_ps(avx_p2, pX2, pR2));
+                __m256 pSrcCol = _mm256_fmadd_ps(pFx, _mm256_add_ps(pFac1, _mm256_add_ps(pFac2, pFac3)), pU0);
+
+                // float rowLoc = fy * (y * kr + tCoeff[0] * (r2 + 2 * ySquare) + tCoeff[1] * xyMul2) + v0;
+                pFac1 = _mm256_mul_ps(pY, pKR);
+                pFac2 = _mm256_fmadd_ps(pTCoeff[1], p2xy, _mm256_mul_ps(pTCoeff[0], _mm256_fmadd_ps(avx_p2, pY2, pR2)));
+                __m256 pSrcRow = _mm256_fmadd_ps(pFy, _mm256_add_ps(pFac1, pFac2), pV0);
+
+                _mm256_storeu_ps(rowRemapTableRow, pSrcRow);
+                _mm256_storeu_ps(colRemapTableRow, pSrcCol);
+                rowRemapTableRow += vectorIncrement;
+                colRemapTableRow += vectorIncrement;
+
+                // xCamera += invMat[0], yCamera += invMat[3], zCamera += invMat[6]
+                pXCamera = _mm256_add_ps(pXCamera, pXCameraIncrement);
+                pYCamera = _mm256_add_ps(pYCamera, pYCameraIncrement);
+                pZCamera = _mm256_add_ps(pZCamera, pZCameraIncrement);
+            }
+            for(; vectorLoopCount < width; vectorLoopCount++)
+            {
+                Rpp32f z = 1./zCamera, x = xCamera * z, y = yCamera * z;
+                Rpp32f xSquare = x * x, ySquare = y * y, r2 = xSquare + ySquare;
+                Rpp32f xyMul2 = 2 * x * y;
+                Rpp32f kr = (1 + ((((rCoeff[2] * r2) + rCoeff[1]) * r2) + rCoeff[0]) * r2) / (1 + ((((rCoeff[5] * r2) + rCoeff[4]) * r2) + rCoeff[3]) * r2);
```

Comment:
In this case since the AVX side shows 3 fmaf for numerator and 3 fmaf for denominator, can we use the same std::fmaf here?
Similarly copy the same raw C line you finalize on, as a comment above the AVX code - to make it easier for the reader.